GenderPayGap.WebUI/Views/Compare/CompareEmployers.cshtml

```diff
@@ -20,10 +20,35 @@
     </div>
 
     <div class="grid-row">
-        <div class="column-two-thirds">
+        <div class="column-two-thirds govuk-!-margin-bottom-2">
             <a class="link-button" rel="track" data-track-category="Related Link Clicked" href="/actions-to-close-the-gap">How can employers close the gap</a>
-            <br/>
-            <br/>
+        </div>
+        <div class="column-full">

```

Comment:
I think `column-two-thirds` might look nicer here (I find the shorter lines of text easier to read)